test/integration/modules/debug/status.spec.ts

```diff
@@ -14,8 +14,8 @@ describe('modules/status', () => {
     const health = await getHealth(BEE_DEBUG_URL)
 
     expect(health.status).toBe('ok')
-    // Matches both versions like 0.5.3-c423a39c and 0.5.3
-    expect(health.version).toMatch(/^\d+\.\d+\.\d+(-[0-9a-f]+)?$/i)
+    // Matches both versions like 0.5.3-c423a39c, 0.5.3-c423a39c-dirty and 0.5.3
+    expect(health.version).toMatch(/^\d+\.\d+\.\d+(-[0-9a-f]+)?(-dirty)?$/i)
```

Comment:
This technically makes it possible to have `0.5.3-dirty` but I don't see that as an issue. If you do want to solve it that would be 
```suggestion
    expect(health.version).toMatch(/^\d+\.\d+\.\d+(-[0-9a-f]{8}(-dirty)?)?$/i)
```